pkg/cli/doctor.go

```diff
@@ -86,22 +89,12 @@ func runClusterDoctor(cmd *cobra.Command, args []string) (retErr error) {
 	}
 	defer sqlConn.Close()
 
-	rows, err := sqlConn.Query(`
+	stmt := `
```

Comment:
do you need a version check to use `crdb_internal_mvcc_timestamp` here?

Reply:
yes, how do I do that exactly (any similar examples?)

Reply:
One way that I've done this is to first issue a query like `select crdb_internal_mvcc_timestamp from ... limit 1` and see if an `UndefinedColumn` error is returned.

Reply:
Thanks!

Reply:
I realize I have similar problem when using `system.namespace2`. Do I need to worry about it if I only want to support clusters >= 20.1? Going forward, which table are going to keep `system.namespace` or `system.namespace2`?

Reply:
after a cluster hits 20.1 the main namespace table to consider is `system.namespace`, (it will be the one with the schema column)

Reply:
You will have to check that the column `parentSchemaID` exists probably